drivers/scheduler/k8s/specs/nginx/px-nginx-storage.yaml

```diff
@@ -6,14 +15,16 @@ metadata:
 provisioner: kubernetes.io/portworx-volume
 parameters:
    repl: "3"
-   shared: "true"
 ---
 ##### Portworx persistent volume claim
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
    name: px-nginx-pvc
    annotations:
+     px/secret-name: volume-secrets
+     px/secret-namespace: portworx
+     px/secret-key: nginx-secret
```

Comment:
we support encryption for shared volumes?

Reply:
No we don't. This storage class does not have shared flag.